ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,149 @@
-﻿namespace ConsoleApp1
+﻿using Microsoft.VisualBasic;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.Metrics;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Beetroot_HOMEWORK
 {
-	internal class Program
-	{
-		static void Main(string[] args)
-		{
-		}
-	}
+    internal class Program
+    {
+        static void PrintArray(char[] chars)
+        {
+            for (int i = 0; i < chars.Length; i++)
+                Console.Write("'{0}'", chars[i]);
+            Console.WriteLine();
+        }
+        static void Main(string[] args)
+        {
+            string str = "Hello World";
+            string str1 = "Hello world";
+
+            Console.WriteLine(Compare(str, str1));
+            Analyze(str);
+            Console.WriteLine(Sort("CBAcbadfga"));
+
+            char[] chars = DuplicateWithMetod(str);
+            char[] chars2 = DuplicateWithoutMetod(str);
+
+            PrintArray(chars);
+            PrintArray(chars2);
+            LoremIpsum();
+
+            Console.ReadLine();
+        }
+        //create next methods:
+        //Compare that will return true if 2 strings are equal, otherwise false, but do not use build-in method
+        static bool Compare(string str, string str1)
+        {
+            int caunt = 0;
+            if (str.Length==str1.Length)
+            {
+                for (int i = 0; i < str.Length; i++)
+                {
+                    if (str[i] == str1[i])
```

Comment:
if first character is different from another string, is it make sense to continue ?